style.css

```diff
@@ -0,0 +1,1014 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap');
+
+:root {
+  --primary: #272a31;
+  --secondary: #ec5242;
+  --light: #d3d3d3;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Lato', sans-serif;
+}
+
+.material-symbols-outlined {
+  font-variation-settings:
+    'FILL' 0,
+    'wght' 800,
+    'GRAD' 0,
+    'opsz' 48;
+}
+
+/* header */
+
+.account {
+  display: none;
+}
+
+header {
+  align-self: flex-end;
+  width: 100%;
+}
+
+nav {
+  width: 100%;
+  padding: 17.5px 5%;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  background-color: transparent;
+}
+
+nav img,
+.menu-items {
+  display: none;
+}
+
+nav .material-symbols-outlined {
+  font-size: 40px;
+  color: var(--primary);
+}
+
+/* Popup-nav */
+
+.popup-nav {
+  display: none;
+  flex-direction: column;
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+  bottom: 0;
+  background-color: rgba(62, 53, 53, 0.861);
+  backdrop-filter: blur(8px);
+}
+
+.popup-nav-visible {
+  display: flex;
+}
+
+.popup-header {
+  display: flex;
+  justify-content: flex-start;
+}
+
+.popup-menu {
+  display: grid;
+  align-items: center;
+  justify-content: center;
+  margin-top: 5%;
+  width: 100%;
+}
+
```

Comment:
- [ ] The images of `about-summit` section are coming on top of `mobile menu` when you scroll while mobile menu is displaying. Please make sure to fix it either by disabling the scrolling while mobile menu is displaying or fixing the positioning of your images and mobile menu

<img width="368" alt="Screenshot 2023-03-18 at 12 43 13 PM" src="https://user-images.githubusercontent.com/44798044/226093626-3c11f2fe-eef1-40a6-9caa-53240f59e748.png">